be/src/exec/olap_scanner.cpp

```diff
@@ -455,7 +455,6 @@ void OlapScanner::update_counter() {
     COUNTER_UPDATE(_rows_pushed_cond_filtered_counter, _num_rows_pushed_cond_filtered);
 
     COUNTER_UPDATE(_parent->_io_timer, _reader->stats().io_ns);
-    COUNTER_UPDATE(_parent->_read_compressed_counter, _reader->stats().compressed_bytes_read);
```

Comment:
You can't remove this. In some case, _update_realtime_counter is not called and `_reader->stats().compressed_bytes_read` is not zero.